File: app/src/main/java/ru/mamykin/foreignbooksreader/ui/adapters/viewholders/StoreBookViewHolder.java
package ru.mamykin.foreignbooksreader.ui.adapters.viewholders;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;

import butterknife.BindView;
import butterknife.ButterKnife;
import ru.mamykin.foreignbooksreader.R;

public class StoreBookViewHolder extends RecyclerView.ViewHolder {
    @BindView(R.id.tvName)
    public TextView tvBookName;
    @BindView(R.id.tvAuthor)
    public TextView tvBookAuthor;
    @BindView(R.id.tvCategory)
    public TextView tvBookCategory;
    @BindView(R.id.tvPrice)
    public TextView tvBookPrice;
    @BindView(R.id.tvOldPrice)
    public TextView tvBookOldPrice;
    @BindView(R.id.tvRating)
    public TextView tvRating;
    @BindView(R.id.ivCover)
    public ImageView ivCover;

    public StoreBookViewHolder(View itemView) {
        super(itemView);
        ButterKnife.bind(this, itemView);
    }
}

File: app/src/main/java/ru/mamykin/foreignbooksreader/ui/adapters/viewholders/StoreBookViewHolder.kt
package ru.mamykin.foreignbooksreader.ui.adapters.viewholders

import android.support.v7.widget.RecyclerView
import android.view.View
import android.widget.ImageView
import android.widget.TextView

import butterknife.BindView
import butterknife.ButterKnife
import ru.mamykin.foreignbooksreader.R

class StoreBookViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
    @BindView(R.id.tvName)
    var tvBookName: TextView? = null
    @BindView(R.id.tvAuthor)
    var tvBookAuthor: TextView? = null
    @BindView(R.id.tvCategory)
    var tvBookCategory: TextView? = null
    @BindView(R.id.tvPrice)
    var tvBookPrice: TextView? = null
    @BindView(R.id.tvOldPrice)
    var tvBookOldPrice: TextView? = null
    @BindView(R.id.tvRating)
    var tvRating: TextView? = null
    @BindView(R.id.ivCover)
    var ivCover: ImageView? = null

    init {
        ButterKnife.bind(this, itemView)
    }
}